src/openrct2/object/ObjectRepository.cpp

```diff
@@ -47,6 +47,28 @@
 
 using namespace OpenRCT2;
 
+struct ObjectIdentifierHash
+{
+    size_t operator()(const std::string_view& identifier) const
+    {
+        std::string copy = identifier.data();
```

Comment:
This is not a good idea. If string_view is not null terminated, then this will cause issues. You should instead be doing `std::string copy = identifier`.

However this function can just iterate over the string_view. You don't need to copy it to a std::string.